### PR TITLE
general: Started working on completion.

### PR DESCRIPTION
**Pull request by dylanaraps**
TODO:

- [x] bash completion.
    - [ ] Figure out if Makefile is correct.
- [ ] zsh completion
- [ ] fish completion

### CODE REVIEW

**Comment by MindTooth** [comment]
If I have the will in the coming days, I might submit a completion for `fish` if you haven't begin locally.

**Comment by dylanaraps** [comment]
Go ahead :)

File: completions/neofetch-completion.bash [hunk 1]
```diff
@@ -0,0 +1,437 @@
...
+shopt -s nullglob
+
+_neofetch_completions() {
+    local cur prev usage
+
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    prev="${COMP_WORDS[COMP_CWORD-1]}"
+
+    if [[ "$cur" == -* ]]; then
+        usage="$(neofetch -h | awk '/^    --/{printf $1" "}')"
```
Comment (MindTooth) [inline completions/neofetch-completion.bash]: Should this be `--help`?

**Comment by MindTooth** [comment]
Initial work.

[asciicast](https://asciinema.org/a/ZidQ4OzwLkgr7Cq3AlOO7SYX5)

File: completions/neofetch-completion.bash [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+                    "mopidy"
+                    "mpd"
+                    "pogo"
+                    "pragha"
+                    "qmmp"
+                    "quodlibet"
+                    "rhythmbox"
+                    "sayonara"
+                    "spotify"
+                    "Spotify"
```
Comment (MindTooth) [inline completions/neofetch-completion.bash]: Ment to be duped?

Comment (dylanaraps) [reply completions/neofetch-completion.bash]: Nope, will fix

**Pull request closed without merging**